### ISSUE TITLE
RUF004 appears to be a duplicate of B026

### ISSUE THREAD

**Issue by CalumY** [charliermarsh/ruff#3290]
Take the snipped: 

```python 
serializer = self.serializer_class(data=request.data, *args, **kwargs)
```

When running `ruff check .` the following errors are returned for the snippet line: 

```
RUF004 Keyword argument `data` must come after starred arguments
B026 Star-arg unpacking after a keyword argument is strongly discouraged
```

Both the error messages report the same error. I don't believe both should be shown in this case.

Ruff version: 0.0.253

**Comment by charliermarsh** [comment]
I _think_ you're right. `RUF004` flags the keyword arguments, while `B026` flags the star argument, but they are effectively the same rule AFAICT.
